api-samples/downloads/download_manager/service-worker.js

```diff
@@ -0,0 +1,219 @@
+chrome.downloads.setShelfEnabled(false);
```

Comment:
This was actually recently deprecated - could we use the replacement? https://developer.chrome.com/docs/extensions/reference/downloads/#method-setShelfEnabled